lib/xray/middleware.rb

```diff
@@ -91,6 +91,7 @@ def script_matcher(script_name)
       /
         <script[^>]+
         \/#{script_name}
+        (2)?                # Optional 2 for jQuery2 format
```

Comment:
Can you fix the spacing here so that the comment lines up with the other comments below it?